examples/kafka-test/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "glee-example",
+  "version": "0.1.0",
+  "description": "An example of the AsyncAPI Glee framework",
+  "type": "module",
+  "scripts": {
+    "dev": "glee dev",
+    "start": "glee start"
+  },
+  "author": "Fran Mendez",
```

Comment:
🙂 

```suggestion
  "author": "Ruchi Pakhle",
```

Reply:
you are the OG author fran, anyday 😄

Reply:
haha! That's not true. You're the author of this example so feel free to change it!